sdk/Lusid.Sdk.Tests/tutorials/Instruments/FxOptionExamples.cs

```diff
@@ -150,5 +152,110 @@ public void FxOptionPortfolioCashFlowsExample(ModelSelection.ModelEnum model, bo
             var fxOption = InstrumentExamples.CreateExampleFxOption(isDeliveryNotCash);
             CallLusidGetPortfolioCashFlowsEndpoint(fxOption, model);
         }
+
+        /// <summary>
+        /// Lifecycle management (i.e. conservation of PV on cashflow events) for fx option
+        /// Cash-settled case:
+        ///      pv(option) = max(spot - strike, 0) before maturity and
```

Comment:
Believe `pv(option) = max(spot - strike, 0) ` is only true for CTVoM
Did I write analogous comment for EQ option?

Reply:
Ah right, reading test, it's only for CTVoM right now..hmm